feature-libs/asm/components/customer-emulation/customer-emulation.component.html

```diff
@@ -4,6 +4,17 @@
     <label class="cx-asm-uid">{{ customer?.uid }}</label>
   </div>
   <cx-asm-bind-cart></cx-asm-bind-cart>
+  <!-- TODO(CXSPA-3090): Remove feature flag in 7.0 -->
+  <ng-container *cxFeatureLevel="'6.3'">
```

Comment:
[MAJOR REMARK - breaking change]
We're targeting `6.5`, so let's update it to 6.5. Otherwise, our customer that configured their `featureLevel` `'6.3'` and they to bump `@spartacus/asm` package to v6.5 from v6.3 (or v6.4), then they will immediately get this feature enabled.
But the expected behavior is that they should get this feature enabled only when they configure their `featureLevel` to be `'6.5'`.